src/components/calcite-button/calcite-button.tsx

```diff
@@ -155,7 +157,7 @@ export class CalciteButton {
         class={{ [CSS.icon]: true, [CSS.iconStart]: true }}
         flipRtl={this.iconFlipRtl === "start" || this.iconFlipRtl === "both"}
         icon={this.iconStart}
-        scale="s"
+        scale={iconScale}
```

Comment:
From a conversation we had with Bryan, I though that S, M, or L button icons need to be 16px. This is what's on Figma too. 

CC'ing @bstifle